build.gradle

```diff
@@ -99,6 +99,14 @@ def check(Object version, String libVersion, String libName) {
     }
 }
 
+def ignoreExplicitApi = [
```

Comment:
This name is better suited for a method, not for a list of things. I suggest  `DisabledExplicitApiModeProjects` or something shorter.